plugins/example/ahda/client.py

```
@@ -0,0 +1,143 @@
+from fastapi import APIRouter, Request, HTTPException, Form, Query, BackgroundTasks, Body
+from fastapi.responses import HTMLResponse, FileResponse, JSONResponse
+from db import get_ahda_url, store_ahda, get_ahda_os
+import os
+import requests
+from models import RealtimePluginRequest, EndpointResponse
```

Ruff

`models.RealtimePluginRequest` imported but unused

Remove unused import

(F401)